e2e-tests/upgrade-consistency/run

```diff
@@ -53,6 +53,16 @@ main() {
     compare_kubectl service/${CLUSTER}-rs0 "-130"
     compare_kubectl statefulset/${CLUSTER}-rs0 "-130"
 
+    # test 1.4.0
+    API="psmdb.percona.com/v1-4-0"
+    kubectl_bin patch psmdb "${CLUSTER}" --type=merge --patch '{
+        "metadata": {"annotations":{ "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"'"$API"'\"}" }}
+    }'
+    wait_for_running "${CLUSTER}-rs0" "1"
+
+    compare_kubectl service/${CLUSTER}-rs0 "-140"
```

Comment:
I think we need to add "compare" files also

Reply:
done